contrib/TestHarness2/test_harness/run.py

```diff
@@ -367,6 +367,13 @@ def run(self):
             command += ['-b', 'on']
         if config.crash_on_error:
             command.append('--crash')
+        if self.long_running:
+            # disable simulation speedup
+            command += ['--knob-sim-speedup-after-seconds=36000']
+            # disable traceTooManyLines Error MAX_TRACE_LINES
+            command += ['--knob-max-trace-lines=1000000000']
+            # set the timeout to 10h
+            config.kill_seconds = 36000
```

Comment:
It's a little weird to change a config value here - I feel like they're supposed to be read only at this point. Could you modify a local variable here instead?

Reply:
yeah, good point.
Just realized we do not need timeout here,
I have updated it to _None_ for long-running tests,
and it will be controlled by the joshua timeout